src/main/java/org/codeforamerica/shiba/output/caf/CoverPageInputsMapper.java

```diff
@@ -96,13 +96,33 @@ private ApplicationInput getUtmSource(Application application, Document document
   private ApplicationInput getPrograms(Application application) {
     List<String> programs = getValues(application.getApplicationData().getPagesData(),
         APPLICANT_PROGRAMS);
+    if (isTribalTANF(application)) {
+      programs.add("TRIBAL TANF");
+    }
+
+    if (isMFIP(application)) {
+      if (!programs.contains("CASH")) {
+        programs.add("CASH");
```

Comment:
Do we need to add this here too? 
https://github.com/codeforamerica/shiba/blob/main/src/main/java/org/codeforamerica/shiba/output/caf/CoverPageInputsMapper.java#L147-L149

Reply:
Actually nevermind - looks like it's just marked for primary applicant based on ticket description

Reply:
Sorry just saw your comments, but yes, should only be needed for the applicant